builtin/fast-export.c

```diff
@@ -593,8 +593,8 @@ static void anonymize_ident_line(const char **beg, const char **end)
 	struct ident_split split;
 	const char *end_of_header;
 
-	out = &buffers[which_buffer++];
-	which_buffer %= ARRAY_SIZE(buffers);
+	out = &buffers[which_buffer];
+	which_buffer ^= 1;
```

Comment:
This change does not take into account the fact that `which_buffer` is used as an index in `buffers`. If the size of `buffer` changes, this "hack" renders the change useless, producing confusion.